Migrations/ArchiveMigration.php

```diff
@@ -53,7 +53,19 @@ public function migrate(Request $request, TargetDb $targetDb)
                     if (!empty($archive['idarchive'])) {
                         $archive['idarchive'] = $this->createArchiveId($targetDb, $archiveTable, $archive['idarchive']);
                         $archive['idsite']    = $request->targetIdSite;
-                        $targetDb->insert($archiveTable, $archive);
+                        try {
+                            $targetDb->insert($archiveTable, $archive);
+                        } catch (\Exception $e) {
+                            if (
+                                $e->getCode() == 23000
+                                || strpos($e->getMessage(), 'Duplicate entry') !== false
+                                || strpos($e->getMessage(), ' 1062 ') !== false
+                            ) {
+                                $this->log('Duplicate entry in ' . $archiveTable . ' table for archiveID:' . $archive['idarchive'] . ' and name:' . $archive['name']);
```

Comment:
It may read a little easier with a couple more spaces.
```suggestion
                                $this->log('Duplicate entry in ' . $archiveTable . ' table for archiveID: ' . $archive['idarchive'] . ' and name: ' . $archive['name']);
```